vividus-facade-jira/src/main/java/org/vividus/jira/JiraFacade.java

```diff
@@ -83,6 +88,16 @@ public JiraEntity getIssue(String issueKey) throws IOException, JiraConfiguratio
         return getJiraEntity(ISSUE, issueKey, JiraEntity.class);
     }
 
+    public void addAttachment(String issueKey, String attachmentName, byte[] attachmentData)
+            throws IOException, JiraConfigurationException
+    {
+        HttpEntity attachmentEntity = MultipartEntityBuilder.create()
+                .addBinaryBody("file", attachmentData, ContentType.MULTIPART_FORM_DATA, attachmentName)
+                .build();
+        jiraClientProvider.getByIssueKey(issueKey).executePost(ISSUE_ENDPOINT + issueKey + "/attachments",
+                List.of(new BasicHeader("X-Atlassian-Token", "no-check")), attachmentEntity);
```

Comment:
what is this header for?

Reply:
This header is mentioned in documentation [here](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-attachments/#api-rest-api-3-issue-issueidorkey-attachments-post) and if we do not provide it the endpoint throws an error, I lost the message but this header disables some verifications